templates/documents/general_list.html

```diff
@@ -2,13 +2,38 @@
 <head>
 {php}html_header_show();{/php}
 {php}$this->assign('GLOBALS', $GLOBALS);{/php}
-
+{php}
+$place_hld = isset($_GET['patient_name'])?$_GET['patient_name']:xl("Patient search or select.");
+$cur_pid = isset($_GET['patient_id'])?$_GET['patient_id']:'';
+if ( isset($_SESSION['isNew']) && !isset($_GET['patient_name']) && isset($_GET['list']) && (int)$cur_pid != 0) {
+    unset($_SESSION['isNew']);
+}
+elseif ($cur_pid == '00') {
+    $_SESSION['isNew'] = true;
+    $cur_pid = '0';
```

Comment:
Regarding '00': is fine as long as you treat as a string or type it otherwise.
Not really odd just thought I had that covered but you're right and I'll find a way.